Misc/NEWS.d/next/Library/2021-06-16-23-57-45.bpo-44431.nA8C9m.rst

```diff
@@ -0,0 +1 @@
+UUID module now supports being called from command line. ex > python -m uuid
```

Comment:
I don't think we need an example in a news entry. I would do way with ``ex > python -m uuid`` .